tests/generic_config_updater/gu_utils.py

```diff
@@ -1,12 +1,15 @@
 import json
 import logging
+import pytest
 from tests.common.helpers.assertions import pytest_assert
 from tests.common.utilities import wait_until
-
+from tests.common.config_reload import config_reload
 
 logger = logging.getLogger(__name__)
 
 CONTAINER_SERVICES_LIST = ["swss", "syncd", "radv", "lldp", "dhcp_relay", "teamd", "bgp", "pmon", "telemetry", "acms"]
+DEFAULT_CHECKPOINT_NAME = "test"
+YANG_IGNORED_OPTIONS    = "-i /FEATURE -i /QUEUE -i /SCHEDULER"
```

Comment:
In the future if someone is testing these 3 tables, they might not notice they are getting ignored from validation. Can we have this as an optional argument passed to the rollback and apply-patch?

Reply:
Currently all tests can only run with these 3 tables ignored. These options can be removed after yang is fixed. These options will be removed at that time.

Reply:
Offline discussion. The ignored 3 tables are intentionally ignored for all calls since they affect all test cases equally.